deployment/v3/mosip/pms-migration-utility/README.md

```diff
@@ -1,6 +1,15 @@
 # Partner Management Service 115-120 Migration Utility
 
 ## Install
-```
-$ ./install.sh
-```
+* Run `./install.sh` to deploy pms-migration-utility.
+  ```
+  $ ./install.sh
+  ```
+* The script will prompt to run pms-migration-utility as cronjob,
+  If provided `Y` then the script will deploy pms-migration-utility as cronjob otherwise script will deploy pms-migration-utility as job.
+* For cronjob, the script will prompt to provide hour(X) to run cronjob in every X hr.
```

Comment:
For Cronjob, the script will prompt to provide hour (X) to run the cronjob in every X hr.